.github/workflows/build-nightly-release.yml

```diff
@@ -0,0 +1,70 @@
+name: Nightly Build
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1-5"
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-core:
+    name: Build core
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: ./.github/actions/build-core
+        with:
+          profile: release
+
+  build-pruntime:
+    name: Build pruntime
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: ./.github/actions/build-pruntime
+
+  publish:
+    needs: [build-core, build-pruntime]
+    name: Pack and publish
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+        with:
+          path: ./dist
+
+      - name: Publish
+        uses: softprops/action-gh-release@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag_name: nightly-${{ steps.date.outputs.date }}
+          prerelease: true
+          body: Nighty build
+          files: |
+            ./dist/pruntime-binaries/pruntime
+            ./dist/core-blockchain-binaries/phala-node
+            ./dist/core-blockchain-binaries/pherry
+            ./e2e/res/system.contract
+            ./e2e/res/log_server.contract
+            ./e2e/res/sidevm_deployer.contract
+            ./e2e/res/tokenomic.contract
+
+      - name: Delete older nightly releases
+        uses: Phala-Network/delete-older-releases@master
+        with:
+          keep_latest: 10
```

Comment:
How about keeping 30 days? Not sure if it's too large. By keeping 30 days, any script can rely on the nightly for at lease one month.

Reply:
OK